apko/private/apko_run.bzl

```diff
@@ -46,9 +48,14 @@ def _impl(ctx):
         is_executable = True,
     )
 
+    additional_files = []
+    for data in ctx.attr.data:
+        if ApkoConfigInfo in data:
+            additional_files += data[ApkoConfigInfo].files.to_list()
```

Comment:
Bazel best practices ask for evaluating this as a depset from day one: 
  
 https://bazel.build/extending/depsets
 
 it's you should build a list of depset's and than wrap it as a depset in line 58

Reply:
Done